.github/workflows/ci.yml

```diff
@@ -50,7 +50,7 @@ jobs:
         with:
           project-token: ${{ secrets.CODACY_PROJECT_TOKEN }}
           coverage-reports: coverage.xml
-        if: ${{ matrix.python-version == '3.7' }}
+        if: ${{ matrix.python-version == '3.7' }} && (github.ref == 'refs/heads/master') }}
```

Comment:
Are you certain that this is the correct syntax? I'd expect the following to be wanted.

```
${{ (matrix.python-version == '3.7') && (github.ref == 'refs/heads/master') }}
```

Reply:
Fixed ;)